### PR TITLE
update README with latest version number

### PR DESCRIPTION
**Pull request by leckie-chn**

### CODE REVIEW

**Comment by QuanluZhang** [comment]
please update all the places that use v1.2

**Comment by chicm-ms** [comment]
there are a few `git clone -b ` commands in readme also need to be updated.

**Comment by ultmaster** [comment]
There is also `v1.2` in `docs/conf.py`.

File: README_zh_CN.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ NNI 管理自动机器学习 (AutoML) 的 Experiment，**调度运行**由调优
 * 想要更容易**实现或试验新的自动机器学习算法**的研究员或数据科学家，包括：超参调优算法，神经网络搜索算法以及模型压缩算法。
 * 在机器学习平台中**支持自动机器学习**。
 
-### **NNI v1.2 已发布！ &nbsp;[<img width="48" src="docs/img/release_icon.png" />](#nni-released-reminder)**
+### **NNI v1.3 已发布！ &nbsp;[<img width="48" src="docs/img/release_icon.png" />](#nni-released-reminder)**
```
Comment (squirrelsc) [inline README_zh_CN.md]: don't change zh_CN files directly. update it in crowdin.com .

**Pull request merged**